kong/db/declarative/import.lua

```diff
@@ -256,63 +416,14 @@ local function load_into_cache(entities, meta, hash)
     end
     local schema = dao.schema
 
-    -- Keys: tag_name, eg "admin"
-    -- Values: dictionary of keys associated to this tag,
-    --         for a specific entity type
-    --         i.e. "all the services associated to the 'admin' tag"
-    --         The ids are keys, and the values are `true`
-    local taggings = {}
-
-    local uniques = {}
-    local page_for = {}
-    local foreign_fields = {}
-    for fname, fdata in schema:each_field() do
-      local is_foreign = fdata.type == "foreign"
-      local fdata_reference = fdata.reference
-
-      if fdata.unique then
-        if is_foreign then
-          if #db[fdata_reference].schema.primary_key == 1 then
-            insert(uniques, fname)
-          end
-
-        else
-          insert(uniques, fname)
-        end
-      end
-      if is_foreign then
-        page_for[fdata_reference] = {}
-        foreign_fields[fname] = fdata_reference
-      end
-    end
-
-    local keys_by_ws = {
-      -- map of keys for global queries
-      ["*"] = {}
-    }
-    for id, item in pairs(items) do
-      -- When loading the entities, when we load the default_ws, we
-      -- set it to the current. But this only works in the worker that
-      -- is doing the loading (0), other ones still won't have it
-
-      yield(true, phase)
-
-      assert(type(fallback_workspace) == "string")
-
-      local ws_id = ""
-      if schema.workspaceable then
-        local item_ws_id = item.ws_id
-        if item_ws_id == null or item_ws_id == nil then
-          item_ws_id = fallback_workspace
-        end
-        item.ws_id = item_ws_id
-        ws_id = item_ws_id
+    for _, item in pairs(items) do
+      if not schema.workspaceable or item.ws_id == null or item.ws_id == nil then
+        item.ws_id = default_workspace_id
       end
 
-      assert(type(ws_id) == "string")
+      assert(type(item.ws_id) == "string")
```

Comment:
we did the assertion earlier.

Reply:
Do you mean `default_workspace_id`? and `item.ws_id` may not be assigned to it.